#std_msgs/Header header
int32 id
JointInfo[] joints 
